README.md

```diff
@@ -6,8 +6,8 @@ UI component demos deployed here: https://nitro-storybook.netlify.app/
 
 ## Packages
 
-- package `site` contains the web UI ([readme](./packages/site/README.md))
-- package `nitro-rpc-client` contains a typescript library for communicating with go-nitro's RPC api. ([readme](./packages/nitro-rpc-client/readme.md))
```

Comment:
These links are currently broken. Also, no need to link directly to readme, the readme is displayed when you navigate to the subdirectory.